src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <!-- <link rel="preconnect" href="https://fonts.googleapis.com">
+         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+         <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" 
+           rel="stylesheet"> -->
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header_logo-_link" href="#home">
+        <img src="./images/logo.png"
+          alt="MoYo"
+          class="header_logo-img">
+      </a>
+      <nav class="nav">
```

Comment:
```suggestion
      </a>
      
      <nav class="nav">
```
don't forget to add space between one-level elements